contrib_bots/bots/translate/Language_Codes.md

```diff
@@ -0,0 +1,57 @@
+Use this document to refer to correct language codes:
```

Comment:
Your table seems to be a little messed up in the preview. The Markdown doesn't seem to have been parsed. You might want to fix it.